tutor_webhook_receiver/patches/local-docker-compose-jobs-services

```diff
@@ -7,7 +7,7 @@ webhook_receiver-job:
       - DB_MIGRATION_PASS={{ WEBHOOK_RECEIVER_DB_PASSWORD }}
       - DB_MIGRATION_HOST={{ MYSQL_HOST }}
       - DB_MIGRATION_PORT={{ MYSQL_PORT }}
-      - DB_MIGRATION_OPTIONS={{ WEBHOOK_RECEIVER_DB_MIGRATION_OPTIONS }}
+      - DB_MIGRATION_OPTIONS='{{ WEBHOOK_RECEIVER_DB_MIGRATION_OPTIONS|tojson }}'
```

Comment:
This breaks when I run it locally with the default value for `DB_MIGRATION_OPTIONS`. I don't think we need to convert it to JSON in docker-compose files.

Reply:
It breaks inside the webhook_receiver service:
```
  File "/webhooks/./webhook_receiver/settings/production.py", line 37, in <module>
    OPTIONS=env.json('DB_MIGRATION_OPTIONS', default=None),  # noqa: F405
  File "/webhook_receiver/venv/lib/python3.8/site-packages/environ/environ.py", line 233, in json
    return self.get_value(var, cast=json.loads, default=default)
  File "/webhook_receiver/venv/lib/python3.8/site-packages/environ/environ.py", line 394, in get_value
    value = self.parse_value(value, cast)
  File "/webhook_receiver/venv/lib/python3.8/site-packages/environ/environ.py", line 454, in parse_value
    value = cast(value)
  File "/usr/local/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

Reply:
Yes the single list option would work.